content/tools/index.njk

```diff
@@ -1,24 +1,59 @@
 ---
 title: Tools
 banner: Open Source Tools
-sub: We build tools that millions of developers rely on
+sub: Support tools that you and million of developers rely on
 list_tag: OddTools
 action:
   text: Sponsor our open web contributions »
   url: https://opencollective.com/oddbird-open-source
+goal:
+  title: Support Logical Shorthands in CSS
+  author: miriam
+  sub: Can we get this process unstuck?
+  summary: |
+    The CSS Working Group recently resolved to add a size shorthand for setting
+    both the width and height of an element. Many people asked about using it to
+    set the ‘logical’ `inline-size` and `block-size` properties instead.
+    But ‘logical shorthands’ have been stalled in the working group for years.
+
+    With a little funding, we could make it happen!
+  url: /2025/03/20/logical-shorthand/
+  sponsor_url: https://opencollective.com/oddbird-open-source/contribute/css-logical-shorthands-86141
+  links: |
+    - **Demo:** [2-Value Logical Shorthands](https://codepen.io/miriamsuzanne/pen/RNwYrEB)
+    - **Progress:** Follow along in [Mia’s public notebook](https://css.oddbird.net/logical/research/)
+    - **Article:** [How do we move logical shorthands forward?](/2025/03/27/logical-options/)
+    - **Article:** [Why inline-size but padding-inline?](https://css.oddbird.net/logical/research/#why-inline-size-but-padding-inline)
+    - **Side Quest:** Should there be physical counterparts for 2-value properties?
+      Let us know what you think on [Mastodon](https://front-end.social/@oddbird) 
+      or [Bluesky](https://bsky.app/profile/oddbird.dev)!
+  image:
+      src: blog/2025/closed.jpg
```

Comment:
I think a screenshot of the demo might work better here?

Reply:
I used this screenshot

![image](https://github.com/user-attachments/assets/1a76256b-8356-4da2-b887-4a734f8ecd62)

but we can switch to the email form demo if that makes more sense. (I added s screenshot of that demo as well just in case. We can remove it if we don't use it.)

Reply:
either one works for me!